stdlib/kokorawr/kokorawr/functions/services/scores.js

```diff
@@ -1,39 +1,41 @@
 const lib = require('lib')({token: process.env.STDLIB_TOKEN});
 
+const winInc = {'bs': 31, 'c4': 23, 'ddg': [1, -5], 'mm': 37, 'rps': 3, 'ttt': 11}
+
 /**
 * Leaderboard information for teams!!!
 * @param {boolean} post if you're trying to store information
-* @param {object} store information to store. Format should be {name: str, info: gameInfo}
+* @param {integer} team which team should increase their score
+* @param {string} game which game was just played
+* param {integer} index which score to use
```

Comment:
Needs the @param

Reply:
The values in score for the slack message do not get updated for the individual game, so there is likely some issue. You can probably clean up the code for that too. It's probably in the commands/scores.js though.
<img width="385" alt="screen shot 2018-01-05 at 4 47 47 pm" src="https://user-images.githubusercontent.com/9023483/34634524-b6067ea0-f239-11e7-877a-db9d31cb92b8.png">

Reply:
Don't forget to do a full test. All local and on Slack before asking for a review.